docs/source/conf.py

```diff
@@ -128,7 +128,7 @@ def __div__(self, other):
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
-exclude_patterns = []
+exclude_patterns = ["tutorials1.rst", "tutorials2.rst", "00-intro.rst"]
```

Comment:
perhaps just for clarity you can rename `tutorials1.rst` -> `tutorials_header.rst` and `tutorials2.rst` -> `tutorials_footer.rst` or something like that